### ISSUE TITLE
all kinds of bugs ...

### ISSUE THREAD

**Issue by zhhtu** [Guanghan/ROLO#22]
hello everyone,
I'm trying to make the model run, but currently there's all kinds of bugs. The first challenge is it is the most unclear README file I've ever found. There's no clue where to start, what is needed, data path changing or anything. I'm wondering is there anyone ever re-product the performance in the paper. Basically, I've just read the pre-trained results with so-called DEMO-file after several days debugging. Not even to mention the code does not support PYTHON 3 tensorflow1.0 or even OpenCV3.0. Also, files are missing (heat results file), file locations are also not mentioned in the README file. 
While, I decided to give up and move on with other papers. It's quite badly documented and the demo results seem even not fair.

**Comment by njoye** [comment]
Currently standing at the same point as @huanhuanxuxu. Did you find an alternative that you were able to get running ?

**Comment by SMZCC** [comment]
Do agree 😭

**Comment by milliyang** [comment]
waste of times...

**Comment by sahandb** [comment]
I agree with these guys too, @Guanghan why you don't respond to our comment!

**Comment by sahandb** [comment]
I just want to develop the code and modify this repository for tensorflow 2.x and python 3.x, but you don't even give us any chance for doing it with this README and these bugs!
